port_name: "/dev/ttyUSB0"
baud_rate: 1000000
min_motor_id: 1
max_motor_id: 4
update_rate: 100
protocol_version: 1.00

xtion_head_lower_tilt_joint_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_lower_tilt_joint
    joint_speed: 2.0
    motor:
        id: 3
        zero_tick: 1831

xtion_head_pan_joint_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_pan_joint
    joint_speed: 2.0
    motor:
        id: 2
        zero_tick: 2124

xtion_head_upper_tilt_joint_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_upper_tilt_joint
    joint_speed: 2.0
    motor:
        id: 1
        zero_tick: 1665
